keep-ui/app/(keep)/workflows/builder/editors.tsx

```
@@ -355,334 +347,377 @@ function WorkflowEditorV2({
+                              onClick={addFilter}
+                              size="xs"
+                              className="ml-1 mt-1"
+                              variant="light"
+                              color="gray"
+                              icon={FunnelIcon}
+                            >
+                              Add Filter
+                            </Button>
+                          </div>
+                          {properties.alert &&
+                            Object.keys(properties.alert as {}).map(
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)